packages/components/src/RichTextEditor/utils/commands/fixtures/helpers.ts

```diff
@@ -5,14 +5,15 @@ import {
   Transaction,
 } from "prosemirror-state"
 import { findChildrenByType } from "prosemirror-utils"
+//  eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Does this comment need to exist? Doesn't seem to apply to other files

Reply:
I think it's cause it's not a spec file that `eslint` is complaining? Not sure